content/java/developer-guide/concepts.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Concepts"
+date: 2017-07-13T22:58:22-04:00
+draft: false
+mainpage: false
+projectpage: false
+projectslug: "java"
+projectname: "WorldWind Java"
+layout: "project"
+---
+
+## Concepts
+
+WorldWind is a collection of components that interactively display 3D geographic information within Java applications. Applications use WorldWind by placing one or more WorldWindow objects in their user interface. That WorldWindow provide the 3D geographic context for the application's information and behaviors.
+
+WorldWind components are extensible. The API is defined primarily by interfaces, so components can be selectively replaced by alternative components. Concrete classes can also be replaced or extended. Extensibility is a fundamental objective of WorldWind.
+
+In addition to WorldWindow, there are several major WorldWind interfaces, all shown in the following diagram.
+
+![WorldWind Java Interface Diagram](/img/java/worldwinddevguideillustrations.png)
```

Comment:
I must have grabbed the thumbnail or a different version, this is fixed in 6c8589263a5